pandas/tests/sparse/test_series.py

```diff
@@ -1360,4 +1360,4 @@ def test_numpy_func_call(self):
                  'argmin', 'argmax', 'min', 'max']
         for func in funcs:
             for series in ('bseries', 'zbseries'):
-                getattr(np, func)(getattr(self, series))
+                getattr(np, func)(getattr(self, series)._values)
```

Comment:
or just move argmax/argmin out of the for loop for now to test separately with a catch of the deprecation warning